packages/11ty/_includes/web-components/image-sequence/index.js

```diff
@@ -280,56 +321,115 @@ class ImageSequence extends LitElement {
     }
   }
 
-  hideAllImages() {
-    this.images.forEach((element) => {
-      element.classList.remove('visible')
-    })
-
-  }
-
   hideOverlays() {
     this.querySelectorAll('.overlay').forEach((element) => {
       element.classList.remove('visible')
     })
   }
 
-  hideImage(imageElement) {
-    imageElement.className = ''
+  /**
+   * @function nextImage
+   * @param {Integer} n Number of steps between start index and end index
+   * 
+   * Set the sequence image to the index `n` steps after the current index
+   */
+  nextImage(n=1) {
+    const newIndex = this.index + n >= this.imageCount
+      ? this.index + n - this.imageCount
+      : this.index + n
+    this.index = newIndex
   }
 
-  hideLoadingOverlay() {
-    this.querySelector('.overlay').classList.remove('visible')
+
+  /**
+   * @function _draw
+   * 
+   * Performs drawing operations against `this.context`
+   **/
+  _draw(image) {
+    this.context.drawImage(image,0,0)
   }
 
   /**
-   * Set the sequence canvas to the index `n` steps after the current index
+   * @function _paintCanvas
+   * @param {ImageBitmap} - image - image to paint 
    * 
-   * @param {Integer} n Number of steps between start index and end index
-   */
-  nextCanvas(n=1) {
-    const newIndex = this.index + n >= this.totalCanvases
-      ? this.index + n - this.totalCanvases
-      : this.index + n
-    this.index = newIndex
+   * Paints the `canvas` element with the image from this.index
+   **/
+  _paintCanvas(image) {
+    if (!this.canvasRef.value) {
+      return
+    }
+
+    if (!this.context) {
+      this.context = this.canvasRef.value.getContext('2d')
+    }
```

Comment:
Use logical nullish assignment
```suggestion
    this.context ??= this.canvasRef.value.getContext('2d')
```